Hunter/Assets/Scripts/Model/Entities/Rabbit.cs

```diff
@@ -15,14 +15,17 @@ public Rabbit() : base()
             BodyRadius = 0.25f; // If size in unity = 3,
                                 // then divide it by 12 and
                                 // You'll get BodyRadius
+            BodySeekRadius = 3; // if seek radius in AnimalGizmos
+                                // is 5, then divide it by 1,6
+                                // and You'll get BodySeekRadius
```

Comment:
Through experimentation, I realized that We have this relationship between Gizmos in Unity and Our metrics